keras_cv/models/vgg19.py

```diff
@@ -68,112 +81,145 @@ def VGG19(
             the output of the model will be a 2D tensor.
         - `max` means that global max pooling will
             be applied.
-      classifier_activation: A `str` or callable. The activation function to use
+      classifier_activation:`str` or callable. The activation function to use
         on the "top" layer. Ignored unless `include_top=True`. Set
         `classifier_activation=None` to return the logits of the "top" layer.
         When loading pretrained weights, `classifier_activation` can only
         be `None` or `"softmax"`.
       name: (Optional) name to pass to the model.  Defaults to "VGG19".
-
     Returns:
       A `keras.Model` instance.
     """
-    if weights and not tf.io.gfile.exists(weights):
-        raise ValueError(
-            "The `weights` argument should be either `None` or the path to the "
-            "weights file to be loaded. Weights file not found at location: {weights}"
+
+    def __init__(
+        self,
+        input_tensor=None,
+        include_rescaling=True,
```

Comment:
Let's keep these two `include_` arguments as the first two, and not include defaults.

Reply:
Fixed, Thank you for guiding through the minor changes. Kindly please review and let me know if any changes to be made.

Reply:
Please remove the defaults for `include_rescaling` and `include_top`

e.g.

```
self,
include_rescaling,
include_top,
classes=None,
...
```